docs/pim/products.md

```diff
@@ -46,6 +46,13 @@ An example of an attribute group can be dimensions (length, width, height).
 
 You can assign both whole attribute groups or individual attributes to a product type.
 
+!!! note "Attribute translations"
+
+    Product attributes are not translatable.
+    Unlike content fields, product attribute names and values cannot differ between languages.
+
+    To help mitigate this limitation, for the information that is intended to be displayed, consider using TextLine fields for short text, RichText fields for longer text that may require formatting, and product attributes for precise product properties or specifications.
```

Comment:
I wouldn't use the word "limitation".

Attribute values are not translatable, because they are for properties of the product - which remain the same regardless of the language.

For all the things that need to be translatable (marketing content: descriptions, names, etc.) people should use content fields.